<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>505</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>505</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>拼图</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics.qrc">
    <normaloff>:/qrc/puzzle.ico</normaloff>:/qrc/puzzle.ico</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>331</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton01">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.1.png</normaloff>:/qrc/1.1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton02">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.2.png</normaloff>:/qrc/1.2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton03">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.3.png</normaloff>:/qrc/1.3.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton04">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.4.png</normaloff>:/qrc/1.4.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton05">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.5.png</normaloff>:/qrc/1.5.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton06">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.6.png</normaloff>:/qrc/1.6.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton07">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.7.png</normaloff>:/qrc/1.7.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton08">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.8.png</normaloff>:/qrc/1.8.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton09">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.9.png</normaloff>:/qrc/1.9.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton10">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.10.png</normaloff>:/qrc/1.10.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton11">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.11.png</normaloff>:/qrc/1.11.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton12">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.12.png</normaloff>:/qrc/1.12.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton13">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.13.png</normaloff>:/qrc/1.13.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton14">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.14.png</normaloff>:/qrc/1.14.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton15">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/1.15.png</normaloff>:/qrc/1.15.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton16">
      <property name="styleSheet">
       <string notr="true">border-top: 1px transparent;
    border-bottom: 1px transparent;
    border-right: 1px transparent;
    border-left: 1px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/qrc/0.16.png</normaloff>:/qrc/0.16.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonShow">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;已用时：00:00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRestart">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>重排</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelect">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>51</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>选图</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
